crates/agglayer-certificate-orchestrator/src/lib.rs

```diff
@@ -197,22 +197,24 @@ where
     ///             Ok(CertifierOutput {
     ///                 new_state: LocalNetworkStateData::default().into(),
     ///                 network: NetworkId::new(0),
-    ///                 certificate: Certificate::new_for_test(network_id, height),
```

Comment:
I think that your fmt isn't configured properly. It should ignore code blocks here

Reply:
fixed, seems to be due to the empty line 234 introduced in `main`, it does not have `///` in the middle of this rustdoc